doc/release-notes/1.5.0.txt

```diff
@@ -84,6 +84,13 @@ Vital Bindings
  * Moved python bindings associated with config from sprokit.pipeline to
    vital.config.
 
+Vital Config
+
+ * Extended the application_config_file_paths() function to list config
+   file paths for both the application name provide and for KWIVER-provided
```

Comment:
Something seems wrong here (grammatically). Did you mean "application name *provide**d***"?